Framework/Algorithms/inc/MantidAlgorithms/UnwrapMonitorsInTOF.h

```diff
@@ -0,0 +1,53 @@
+#ifndef MANTID_ALGORITHMS_UNWRAPMONITORSINTOF_H_
+#define MANTID_ALGORITHMS_UNWRAPMONITORSINTOF_H_
+
+#include "MantidAlgorithms/DllConfig.h"
+#include "MantidAPI/Algorithm.h"
+#include "MantidAPI/MatrixWorkspace_fwd.h"
+
+namespace Mantid {
+namespace Algorithms {
+
+/** UnwrapMonitorsInTOF : Handles workspaces which only contain monitors
+ *  and have data recorded which spills over from the previous frame.
+ *  This can occur when dealing with different time regimes for detectors
+ *  and monitors.
+
```

Comment:
This mentions workspaces that only contain monitors, but I think it checks and only moves the monitors.

I don't think the tests check that only monitors are moved, perhaps that should be verified too?